proposals/accessibility/README.md

```diff
@@ -0,0 +1,260 @@
+    These are usually based on metadata on user interface items, such as image alt text or button descriptions.
+* **ARIA (Accessible Rich Internet Application) Roles** : A set of roles and attributes to make the web more accessible. 
+
+### Related Reading
+
+Here are some related materials to read on about the use of accessibility in various systems.
+None of these are required reading, but may help provide context.
+
+* [Apple Accessibility](https://www.apple.com/ca/accessibility/)
+  * [Catch up on accessibility in SwiftUI](https://developer.apple.com/wwdc24/10073)
+  * [Improving the Accessibility of RealityKit Apps](https://developer.apple.com/documentation/realitykit/improving-the-accessibility-of-realitykit-apps)
+* [Microsoft Accessibility](https://www.microsoft.com/en-us/accessibility)
+* [Web Accessibility Guidelines](https://www.w3.org/TR/WCAG21/)
+* [ARIA Roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
+* [Unity 3D Accessibility](https://docs.unity3d.com/2023.2/Documentation/ScriptReference/UnityEngine.AccessibilityModule.html)
+
+## Details 
+
+### API Schema
+
+We propose a new Applied API schema that will allow any Prim to provide descriptions for Audio Description tools. 
+
+We include the following attribute groupings, with the `accessibility` namespace prefix.
+
+* `label`: The primary description string that is presented to the user. 
+* `extendedDescription` : An extended description string that may provide more detail than the primary label.
+* `importance` : Sets the importance of this group so tools may prioritize what to surface to a user. Options are `high`, `standard`, `low` with a default of `standard`.
+
+All three attributes are based on existing standard names in accessibility frameworks, and in consultation with multiple accessibility experts.
+
+Given the example in the summary, repeated here
+
+```python
+def Mesh "Cube" (
+    prepend apiSchemas = ["AccessibilityAPI"]
+) {
+    string accessibility:label = "Luxo, Jr"
+    string accessibility:extendedDescription = "The lamp has round base with two sections above it that may be adjusted. It has a conical head with a lightbulb inside. It likes to chase inflatable balls"
+    token accessibility:important = "standard"
+}
+```
+
+Only the label is required, and so the shortest form of this may be as simple as
+
+```python
+def Mesh "Cube" (
+    prepend apiSchemas = ["AccessibilityAPI"]
+) {
+    string accessibility:label = "A sentient lamp with an adjustable body and cone head"
+}
+```
+
+### Attribute Purposes
+
+It can be useful to have multiple types of descriptions, represented as purposes.
+For example, a primary description may choose to describe a general appearance, but a secondary one may choose to describe a specific detail.
+
+For example, a user may want to have a short summary of the object, but then ask details about its size.
+
+We propose that these be represented as purposes on the attribute.
```

Comment:
@dgovil It seems like purposes could be represented as instances of multi apply schemas. I'm curious if you see any drawbacks to that.

Reply:
Would that be like what you do here?
```
apiSchemas = ["SemanticsCaptionsAPI:summary",
                  "SemanticsCaptionsAPI:skills"]
```

In that case, how would the default unprefixed purpose be declared? Would it just be `SemanticsCaptionsAPI` without the suffix as well?

Reply:
Not being able to have instance-less multi apply schemas is something we have run into before.  Our best recommendation under the current system would be to have a canonical instance-- like `default`.  ie-- `SemanticsCaptionsAPI:default`.

@crevilla might have some thoughts here about the complexities of allowing instance-less multi-apply schemas.  The risk I see is a greater potential for namespace collisions.

Reply:
I hadn't thought about it before, but I don't have a strong argument against allowing instance-less multi-apply schemas. An "instance-less" instance of a multi apply schema wouldn't be much different than a single apply schema and isn't really much different in regards to namespace collisions than single apply schemas. 

We already validate against using property base names as instance names for multi-apply schema (during calls to Apply only, not in general). E.g. "includes" is not a valid instance name for applying CollectionAPI since all applied CollectionAPI instances have a collection:<instanceName>:includes property. I don't see an immediately obvious issue that an empty instance would bring that we aren't already accounting for.

All that said, I'm not sure if an empty instance name is strictly better than a canonical "default" instance.

Reply:
Thanks @crevilla!  This is great info.

I think I would say that allowing empty instance names would be helpful when--
a) converting a single apply schema to a multiapply schema but still supporting legacy content.
b) making it clear to both OpenUSD and users what the canonical instance is. Otherwise, it's just a convention.

Reply:
Just picking this comment thread up again, 

Perhaps it makes sense to just have the default purpose be explicitly `accessibility:label:default` rather than be implicit? Then there's no logic that needs to be thought about, and its fairly straightforward to do a 1:1 mapping at the expense of a little verbosity?